crawler/spiders/nhentai.py

```diff
@@ -61,7 +65,7 @@ def crawl(self, item):
 
 
 def convert_url(url):
-    match_type = re.search(r"jpg|png|gif$", url)
+    match_type = re.search(r"jpg|png|gif|webp$", url)
```

Comment:
Have you tested generating an epub file containing webp?

Do I need to change the version in comicepub to 3.3? https://en.wikipedia.org/wiki/EPUB#Version_3.3

https://github.com/moeoverflow/comicepub/blob/ae10de4ed232da308f340842a56e1fc815bcbd55/comicepub/template/standard.opf#L4

Reply:
I've tested with CdisplayEx and it seems fine. But it's better to update the version I think.